index.js

```diff
@@ -0,0 +1,144 @@
+"use strict";
+
+const startPage = document.querySelector(".page-start"),
+    startBtn = startPage.querySelector(".startBtn-start");
+
+const playingPage = document.querySelector(".page-playing"),
+    digitForm = playingPage.querySelector(".form-playing"),
+    digitInput = playingPage.querySelector(".input-playing"),
+    scoreBoard = playingPage.querySelector(".scoreboard-playing");
+
+const endPage = document.querySelector(".page-end"),
+    leaderBoard = endPage.querySelector(".leaderBoard-end"),
+    restartBtn = endPage.querySelector(".restartBtn-end");
+
+const VAILD_DIGIT_LENGTH = 3;
+const HIDING = "hiding";
+
+let wholePage = [startPage, playingPage, endPage];
+
+let ballCount,
+    strikeCount,
+    inningCount;
```

Comment:
여기도 위에서 말씀드린 것처럼 각각 변수를 선언해주시면 좋을 것 같습니다.
```suggestion
let ballCount;
let strikeCount;
let inningCount;
```